package.json

```diff
@@ -1,10 +1,19 @@
 {
   "name": "decanter",
   "description": "Pattern Library and CSS Framework.",
-  "author": "Stanford University",
+  "author": "Stanford Web Services",
   "version": "7.0.0-beta.0",
   "homepage": "https://github.com/SU-SWS/decanter",
-  "license": "MIT",
+  "license": "GPL-3.0-or-later",
+  "keywords": [
+    "CSS",
+    "Decanter",
+    "Stanford",
+    "TailWindCSS"
+  ],
+  "source": "src/decanter.js",
+  "main": "dist/decanter.js",
```

Comment:
Just a question about this - do you mean decanter.css here since we don't have a decanter.js file in dist/?

Reply:
Yeah, that!